docs/pages/application-access/guides/azure.mdx

```diff
@@ -0,0 +1,554 @@
+---
+title: "Protect the Azure CLI with Teleport Application Access"
+description: How to enable secure access to the Azure CLI.
+---
+
+You can use Teleport Application Access to manage the commands that your users
+can run via the Azure CLI. This lets you control access to your
+infrastructure's management APIs using the same RBAC system that you use to
+protect your infrastructure itself.
+
+The Teleport Application Service runs on an Azure VM and proxies requests from
+end users to Azure's APIs. Teleport users can then execute requests via the
+Azure CLI through Teleport as long as one of their roles allows access to an
+appropriate Azure managed identity. 
+
+You can run the Application Service VM in a private network to prevent
```

Comment:
Is this instead of the declaration above "The Teleport Application Service runs on an Azure VM...", or does it refer to an Azure VM in a private network (not publicly accessible), as opposed to a VM in a physical private network like an in-office LAN?

Reply:
I've added some context—this refers to running the VM without exposing any ports to the public internet